scripts/python/tao_build.py

```diff
@@ -451,8 +472,9 @@ def bazel_build(target, flag=""):
             flag += ' --config=disc_mkldnn'
 
         bazel_build(TARGET_TAO_COMPILER_MAIN, flag=flag)
-        bazel_build(TARGET_DISC_OPT, flag=flag)
-        bazel_build(TARGET_DISC_REPLAY, flag=flag)
+        if not args.dcu and not args.rocm:
+            bazel_build(TARGET_DISC_OPT, flag=flag)
```

Comment:
Why disc-opt build is skipped for dcu/rocm? The disc-opt binary is required for disc FileCheck unit tests.